src/amo/pages/Addon/index.js

```diff
@@ -118,10 +119,11 @@ export class AddonBase extends React.Component {
         if (slugIsPositiveID(slug) || addon.slug !== slug) {
           // We only load add-ons by slug, but ID must be supported too because
           // it is a legacy behavior.
+          const url = getAddonURL(addon.slug);
```

Comment:
We don't really need this `const`. You could just say:

```
url: `/${lang}/${clientApp}${getAddonURL(addon.slug)}`
```

on line 126, below.